lib/crypto/packaged.py

```diff
@@ -116,10 +116,8 @@ def _no_sign(src, dest):
     # If this is a local development instance, just copy the file around
     # so that everything seems to work locally.
     log.info('Not signing the app, no signing server is active.')
-    dest_dir = os.path.dirname(dest)
-    if not os.path.exists(dest_dir):
-        os.makedirs(dest_dir)
-    shutil.copy(src, dest)
+    with storage.open(dest, 'w') as destf:
+        shutil.copyfileobj(open(src), destf)
```

Comment:
This seems wrong, and breaks a few tests when remote storage is enabled. The `src` here will be the `file_path` of a `File` instance, which is supposed to be using the storage already. So we should be using `storage.open()` here.